sklearn/linear_model/_glm/glm.py

```diff
@@ -813,33 +815,48 @@ def __init__(
         warm_start=False,
         verbose=0,
     ):
-
         super().__init__(
             alpha=alpha,
             fit_intercept=fit_intercept,
-            family=TweedieDistribution(power=power),
-            link=link,
+            base_loss_class=HalfTweedieLoss,
+            base_loss_params={"power": power},
```

Comment:
Duplicating the path for `power` might lead to bugs. I would not put it in `base_loss_params`, since we use `self.base_loss_params["power"] = self.power` later anyway.

Reply:
Resolved by https://github.com/lorentzenchr/scikit-learn/pull/3.